### DISCUSSION TITLE
Add custom dropdown menus in menu

### DISCUSSION THREAD

**Discussion by BaxAndrei** [NamelessMC/Nameless#2127] [Ideas]
add custom dropdown menus in menu with permissions too

**Comment by itzarty** [comment]
@baxandrei It's very good idea, but it is possible already. In administration is template editor, there you can without problem make your own dropdown.
But I know, everyone cant know HTML &  Bootstrap so I also think it is good idea.
Sorry for my bad English

**Comment by BaxAndrei** [comment]
You are right, but it is easier because there are users here who do not know how to do this. Although we know how to do this, I think it's a good idea for others and it will help.

**Comment by itzarty** [comment]
I think NamelessMC is transforming into better WordPress :P
